pkg/component/etcdcopybackupstask/etcdcopybackupstask_test.go

```diff
@@ -44,7 +44,7 @@ var _ = Describe("EtcdCopyBackupsTask", func() {
 		values              *Values
 		etcdCopyBackupsTask Interface
 
-		notFoundErr = apierrors.NewNotFound(schema.GroupResource{}, "etcdcopybackupstask")
+		notFoundErr error = apierrors.NewNotFound(schema.GroupResource{}, "etcdcopybackupstask")
```

Comment:
Use `BeNotFoundError()` instead and delete this variable

Reply:
I cannot remove the variable entirely because it is used for the mock client too
`c.EXPECT().Delete(ctx, expected).Return(notFoundErr)`
However, I reverted this change and I'm using  `BeNotFoundError()` instead of `MatchError` now.